src/Common/checkStackSize.h

```diff
@@ -1,5 +1,9 @@
 #pragma once
 
+#include <cstdlib>
+
+size_t getStackSize(void **address_ = nullptr);
```

Comment:
A comment is needed.

(why to pass address by pointer? and what is that address?)